<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sr_cooling_thermocouple_widget</class>
 <widget class="QWidget" name="sr_cooling_thermocouple_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>21</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,1,1">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QECheckBox" name="chkBoxEnable">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCellName">
     <property name="text">
      <string>#CellName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="temperature">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>SRC01-CO-THC1:getTemperature</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QESimpleShape" name="ledEnable">
     <property name="variable" stdset="0">
      <string>$(group):isEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SR-PS-INK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QECheckBox</class>
   <extends>QCheckBox</extends>
   <header>QECheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
